omod/src/main/java/org/openmrs/module/registrationapp/page/controller/RegisterPatientPageController.java

```diff
@@ -56,6 +61,17 @@ public void addModelAttributes(PageModel model, Patient patient, AppDescriptor a
         model.addAttribute("enableOverrideOfAddressPortlet",
                 Context.getAdministrationService().getGlobalProperty("addresshierarchy.enableOverrideOfAddressPortlet", "false"));
         model.addAttribute("breadcrumbOverride", breadcrumbOverride);
+
+        // Relationship Types to exclude (Doctor/Patient and Supervisor/Supervisee)
```

Comment:
Couldn't find it on my code base. I might have seen it on some older releases of some module.

If you want I can remove Doctor/Patient and Supervisor/Supervisee relationships from exclusion list.